dts/bindings/sensor/ti,ina23x-common.yaml

```diff
@@ -34,10 +34,10 @@ properties:
       while keeping a good measurement resolution. The units are in uA/LSB
       so that low maximum currents can be measured with enough resolution.
 
-  rshunt-milliohms:
+  rshunt-micro-ohms:
```

Comment:
The dash looked odd here to me.  So I checked other bindings, I didn't find any of the form "prefix-unit".  It's always "microvolt", "milliamp", etc.  And a handful of "mv" and a single "uv".

Reply:
So I followed what I was told to be the convention: https://github.com/zephyrproject-rtos/zephyr/pull/59861#discussion_r1247812946

Reply:
To your point though I originally did have it without a dash.

Reply:
Indeed it did come up, sorry, I missed that comment.  And dt-schema does say micro-ohm is the standard.  Odd choice, IMHO.  The description of "micro-ohm" being "microohm" is quite perplexing.

```
  "-micro-ohms$":
    $ref: types.yaml#/definitions/uint32-array
    description: microohm
  "-microwatt$":
    $ref: types.yaml#/definitions/uint32-array
    description: microwatt
  "-milliwatt$":
    $ref: types.yaml#/definitions/uint32-array
    description: milliwatt
```

Reply:
No worries, thanks for looking it up though!